src/pokemon_debug.c

```diff
@@ -40,13 +40,13 @@
 #define DEBUG_MON_Y 11
 #define DEBUG_MON_BACK_X 40
 #define DEBUG_MON_BACK_Y 50
-#define DEBUG_ICON_X 158
-#define DEBUG_ICON_Y 90
+#define DEBUG_ICON_X 19//158 + 32
+#define DEBUG_ICON_Y 19//90 + 40
```

Comment:
Are the comments necessary? :eyes:
They don't even seem to be accurate?

Reply:
no, those were the old coordinates for the icon in the bottom right - I'll remove them in the next commit